MatchmakingServer/Program.cs

```diff
@@ -199,4 +199,21 @@ [new Claim(ClaimTypes.Name, playerName),
     return Results.SignIn(claimsPrincipal);
 });
 
+app.MapGet("/stats", (PlayerStore playerStore, ServerStore serverStore, PartyService partyService, Matchmaker matchmaker) =>
```

Comment:
I feel like the Program.cs file is getting quite lengthy. 
It might be time to clean it up and extract how services are set up and how the app is build.

Furthermore, I if an endpoint changes or w/e, you shouldn't have to change the program.cs file. 
I recommend to just make a controller for http request maps. It is just a bit better separation. (imo)

Reply:
Yeah we should definetely clean it up, same for the `App.xaml.cs`. I will create an issue for that, maybe i can clean it up as part of the multi-game refactoring.

I do like minimal api though it is more performant and... minimal :D So for single endpoints i think it makes sense. Would you be fine with putting them into seperate extension methods / classes?

Reply:
But isn't that just the same as adding a controller class? xD

Reply:
Can't you make it like that example? That would be nice. And it is out of the program cs

Reply:
Yes will do that. Also a good time to add some validation to the login endpoint